contrib/libbtrie/src/btrie.c

```diff
@@ -46,7 +46,7 @@ btrie_create()
     tree->free  = NULL;
     tree->start = NULL;
     tree->size  = 0;
-    memset(tree->pools, 0, sizeof(btrie_t *) * BTRIE_MAX_PAGES);
+    memset_s(tree->pools, sizeof(char *) * BTRIE_MAX_PAGES, 0, sizeof(btrie_t *) * BTRIE_MAX_PAGES);
```

Comment:
The whole set of 'safe' functions with the _s suffixes is poorly supported. Clang/GCC does not support `memset_s`...
https://stackoverflow.com/a/50921096

Reply:
What about this?
```suggestion
#ifdef __STDC_LIB_EXT1__
    memset_s(tree->pools, sizeof(char *) * BTRIE_MAX_PAGES, 0, sizeof(btrie_t *) * BTRIE_MAX_PAGES);
#else
    memset(tree->pools, 0, sizeof(btrie_t *) * BTRIE_MAX_PAGES);
#endif
```

Reply:
The `_s` versions are basically Microsoft-only. Plus they are not as safe as they are designed to be. So I tend to not use them.

I'll close this PR.